amy/workshops/models.py

```diff
@@ -75,7 +75,10 @@ class OrganizationManager(models.Manager):
         "software-carpentry.org",
         "datacarpentry.org",
         "librarycarpentry.org",
-        "carpentries.org",  # Instructor Training organisation
+        # Instructor Training organisation
+        "carpentries.org",
+        # Collaborative Lesson Development Training organisation
+        "carpentries.org/community-lessons/",
```

Comment:
Does the `clean_domain` in `OrganizationForm` work with this domain?

Reply:
Yes, the format is covered by this test: https://github.com/carpentries/amy/blob/develop/amy/fiscal/tests/test_organization.py#L56